postr/config.py

```diff
@@ -95,8 +95,10 @@ def update_api_key(api: str, key: str, value: str) -> ConfigParser:
     try:
         config[api][key] = value
         _save_config(config)
+        # pylint: disable=logging-fstring-interpolation
```

Comment:
can you explain what pylint is complaining about here and why it needs to be ignored?

Reply:
Pylint prefers the printf %s style of syntax, rather than string interpolation. I, like you, prefer interpolation